subprojects/gradle/impact/src/main/kotlin/com/avito/impact/plugin/ImpactAnalysisPlugin.kt

```diff
@@ -44,6 +44,11 @@ class ImpactAnalysisPlugin : Plugin<Project> {
             group = "impact-analysis"
             description = "Print modified projects to files. For testing purposes only"
         }
+
+        rootProject.tasks.register<ImpactReportTask>("impactReport") {
+            group = "impact-analysis"
+            description = "Print impact report"
```

Comment:
This misleads me. It will send metrics to a production also.

How about an additional task for debugging with only printing?